### PR TITLE
Geospatial Data Analysis With R

### PR DESCRIPTION
**Pull request by JMParr**

### CODE REVIEW

File: en/lessons/geospatial-data-analysis.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
+---
+title: |
+    Using Geospatial Data to Inform Historical Research in R
+    
+date: 2018-08-03
+authors:
+    - Eric Weinberg
+reviewers:
+	 - Lauren Tilton
```
Comment (mdlincoln) [inline en/lessons/geospatial-data-analysis.md]: Use 4 spaces when indenting the YAML, instead of using tabs.

**Review by mdlincoln — DISMISSED** [review]

File: en/lessons/geospatial-data-analysis.md (hunk 1 above)
Comment (JMParr) [reply en/lessons/geospatial-data-analysis.md]: Thanks. Will do.

**Comment by JMParr** [comment]
One more error:

2:50:26 PM: Destination: /opt/build/repo/_site
2:50:26 PM: Incremental build: disabled. Enable with --incremental
2:50:26 PM: Generating...
2YAML Exception reading
/opt/build/repo/en/lessons/geospatial-data-analysis.md: (<unknown>): could
not find expected ':' while scanning a simple key at line 20 column 1

**Review by JMParr — COMMENTED** [review]
Fixed.

**Review by mdlincoln — CHANGES_REQUESTED** [review]
OK @JMParr, it looks like there's just some link formatting issues to fix now. Travis sends the following:

```
- ./_site/en/lessons/geospatial-data-analysis.html
  *  External link http://programminghistorian.github.io/ph-submissions/lessons/data_wrangling_and_management_in_R failed: 404 No error
  *  External link http://www.datacarpentry.org/r-spatial-data-management-intro/R/intro-to-coordinate-reference-systems failed: 404 No error
  *  External link https://github.com/programminghistorian/ph-submissions/blob/gh-pages/lessons/correspondence-analysis-in-R.md failed: 404 No error
  *  internal image ../images/geospatial-data-analysis/Bar.png does not exist (line 526)
  *  internal image ../images/geospatial-data-analysis/CH1.png does not exist (line 472)
  *  internal image ../images/geospatial-data-analysis/Data2.png does not exist (line 386)
  *  internal image ../images/geospatial-data-analysis/DataLoaded.png does not exist (line 374)
  *  internal image ../images/geospatial-data-analysis/Fit.png does not exist (line 513)
  *  internal image ../images/geospatial-data-analysis/NCSC.png does not exist (line 392)
  *  internal image ../images/geospatial-data-analysis/Plot.png does not exist (line 499)
  *  internal image ../images/geospatial-data-analysis/Ply1.png does not exist (line 564)
  *  internal image /gallery/geospatial-data-analysis.png does not exist (line 185)
- ./_site/en/lessons/index.html
  *  internal image /gallery/geospatial-data-analysis.png does not exist (line 1083)
```

You'll want to edit those external links that are failing. N.b. that link to ph-submissions should point to the published correspondence analysis lesson using `/en/lessons/correspondence-analysis-in-R`

You'll also need to edit the markup for all the figures so that they follow the special syntax detailed here: https://programminghistorian.org/en/author-guidelines#including-images

That should fix most of the broken image links in the lesson right now.

**Comment by JMParr** [comment]
Thanks, Matthew! I'll try to address this tomorrow.

**Comment by JMParr** [comment]
Have a copy that's fixed, less one page from data carpentry referenced in
the footnotes that's been removed. I have emailed the author to ask for an
updated resource.

**Comment by JMParr** [comment]
Got a fixed file ready to go. Looks like it needs an approval on the merge? 
screen shot 2018-08-06 at 10 36 02 am

**Comment by mdlincoln** [comment]
@JMParr you need to make sure that you are on the `geospatial-data` branch when you add and commit the changes. It looks like you switched to the `gh-pages` branch (which is the LIVE version of the site) before making the edits, and I've set the repository to reject any direct commits to `gh-pages`.

**Comment by acrymble** [comment]
@JMParr can you let us know what if any help you need?

**Comment by acrymble** [comment]
@JMParr I don't see a gallery icon here for the lessons page. Can you please make sure you go through the full Editorial Guidelines checklist to make sure you've got everything? 

https://programminghistorian.org/en/editor-guidelines

**Comment by JMParr** [comment]
I have the image. I'd had it added in the image folder, but it appears to
have disappeared. I'll re-add, and I think I just need to add in the
mark-up?  I'm sorry to be struggling so much with this.

**Comment by acrymble** [comment]
It goes in the /gallery directory. There isn't any markup to add as long as you name it following the instructions in the editor's guidelines checklist.

**Comment by JMParr** [comment]
Thanks. I just fixed the sizing and pushed the image. Should be up there
now.

**Comment by acrymble** [comment]
Where are you doing this? Are you trying to commit directly to the gh-pages branch? Make sure you change to the geospatial-data branch before trying to make any changes. Otherwise your changes will be automatically deleted. Screenshots on what you don't want:

bad

And what you do want:

good

**Comment by JMParr** [comment]
I was indeed on the geospatial branch when I pushed the commit.

**Comment by JMParr** [comment]
Here's a view of the change history.

**Comment by acrymble** [comment]
I don't think you have because if you did we would see a list of more commits at the bottom of this thread. Like in this example from @drjwbaker: 
screen shot 2018-08-17 at 07 51 25

**Comment by acrymble** [comment]
The full step-by-step instructions are available on the Wiki for making technical contributions: https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions

I can try to help, but at the minute I'm not getting a clear sense of where you're finding those instructions confusing and therefore where you're going wrong.

**Comment by JMParr** [comment]
I am not sure where things are going wrong either. I've gone back through
the instructions again, as well as my notes from talking to Matthew. I can
send you the gallery image if that's helpful?

**Comment by JMParr** [comment]
I put the image into the gallery folder on my Github desktop and then
pushed the commit.

**Comment by acrymble** [comment]
@JMParr oh ok I didn't realise you were using Github Desktop. I don't use that because I found it confusing. I just use the web GUI. You can email me the gallery image if you want.

**Comment by JMParr** [comment]
Yes, I was advised to use Github Desktop - especially as a newer user of
Github. Didn't want to mess up the site. I'll email you the gallery.

**Comment by mdlincoln** [comment]
@JMParr Sorry to have gone quiet during my move to Pittsburgh! Thanks @acrymble for helping out here. Please let me know if I can be of further help - I'm happy to do another Skype session if that would be easiest. And I'll be on Eastern time, now 👍

**Comment by JMParr** [comment]
Thanks, Matt. I held off because I had a feeling you were busy mid-move.
For some reason, my final corrections don't seem to be pushing up to the
site. Let me see if I can come up with some screenshots. That might help
you identify what I'm missing.

**Comment by JMParr** [comment]
@mdlincoln So, I have made fixes
update on textwrangler
 (including date) the lesson on TextWrangler

**Comment by JMParr** [comment]
@mdlincoln Then I went to Github Desktop to make the commit, and being careful to uncheck the items I did not want to include in the commit.
github

**Comment by JMParr** [comment]
@mdlincoln Here's the commit.
commit

**Comment by JMParr** [comment]
@mdlincoln I then went to the online Github jekyll repository, and I don't see my update. @acrymble has the 2 gallery files - I sent him the one to be used, and followed up with the original (pre re-sizing and gray scale). I'm obviously missing a step here, but am not sure what's going on.

**Comment by mdlincoln** [comment]
@JMParr after you clicked "Commit to **geospatial data**" you also need to *push* the changes so that they're uploaded to GitHub.

**Comment by JMParr** [comment]
@mdlincoln - Thanks. I thought I had. I'll go back and recheck.

**Comment by mdlincoln** [comment]
@JMParr push one more time. It looks like you managed to merge the changes that I made on github, but still haven't pushed your chagnes that include adding the image files.

**Comment by JMParr** [comment]
Matt - it won't let me push again.

**Comment by acrymble** [comment]
I know @mdlincoln has suggested using github desktop, but as someone who like @JMParr absolutely finds github confusing, I might recommend using our technical contribution instructions on the wiki, which uses the github GUI instead. I find it much more intuitive. https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions

**Comment by mdlincoln** [comment]
@JMParr perhaps one more Skype call would be helpful, so I can see what you're seeing on your screen. Perhaps we can find some time this week. I'll email you separately.

**Comment by JMParr** [comment]
@mdlincoln So, one more round of failures. I think this means I forgot to truncate the internal Programming Historian links to /en.... am checking.

**Comment by JMParr** [comment]
@mdlincoln Fixed the markdown errors - there's one code block where the code seems to be showing under "Getting Started" in the preview. Otherwise, it looks better when I look at the preview. [https://deploy-preview-964--ph-dev.netlify.com/en/lessons/geospatial-data-analysis]

**Comment by acrymble** [comment]
I think there's a markdown error on the endnote superscripts in the text:
screen shot 2018-08-30 at 19 31 22

I presume those are meant to be endnote markers (superscripts)?

**Comment by JMParr** [comment]
They probably are. I'll fix them later.

**Review by mdlincoln — APPROVED** [review]

**Comment by mdlincoln** [comment]
@JMParr I went ahead and repaired the footnote,s as well as a bunch of typos. Please take a last look at the preview: https://deploy-preview-964--ph-dev.netlify.com/en/lessons/geospatial-data-analysis

You may squash and merge when ready.

**Comment by JMParr** [comment]
Thanks, @mdlincoln. Looks good. So, it's just a push from Github Desktop
when I squash and merge? I can do it tonight.

**Comment by mdlincoln** [comment]
No, you just click the big green button that says "Squash and merge" on this webpage ⬇️

**Comment by JMParr** [comment]
Thanks! I'll try to do it this afternoon. (First day back teaching.)

**Comment by JMParr** [comment]
@mdlincoln  Squashed and merged.

**Pull request merged**